draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -119,6 +122,44 @@ public void setClickToMinimizeEnabled(boolean enableClickToMinimize) {
   }
 
   /**
+   * Return if touch listener is enable or disable
+   */
+  private boolean isEnableTouch() {
+    return this.enableTouch;
+  }
+
+  /**
+   * Enable or disable the touch listener
+   *
+   * @param enableTouch to enable or disable the touch event.
+   */
+  public void setEnableTouch(boolean enableTouch) {
+    this.enableTouch = enableTouch;
+  }
+
+  /**
+   * Slide the view based on scroll of the nav drawer.
+   * "setEnableTouch" user prevents click to expand while the drawer is moving.
+   * When the slideOffset is bigger than 0.1 and dragView isn't close, set the dragView
+   * to minimized.
+   * It's only possible to maximize the view when @slideOffset is equals to 0.0,
+   * in other words, closed.
+   *
+   * @param slideOffset Value between 0 and 1, represent the value of slide:
+   * 0.0 is equal to close drawer and 1.0 equals open drawer.
+   * @param drawerPosition Represent the position of nav drawer on X axis.
+   * @param width Width of nav drawer
+   */
+  public void slideHorizontally(float slideOffset, float drawerPosition, int width) {
+    float slide = Math.abs(drawerPosition);
+    if (slideOffset > MIN_SLIDE_OFFSET && !isClosed()) {
+      minimize();
+    }
+    setEnableTouch(!(slideOffset <= MIN_SLIDE_OFFSET));
+    ViewHelper.setX(this, width - slide);
+  }
```

Comment:
As I said in my previous comment where I attached you a video with a visual bug, your method implementation should be:

```java

  public void slideHorizontally(float slideOffset, float drawerPosition, int width) {
    float slide = Math.abs(drawerPosition);
    if (slideOffset > MIN_SLIDE_OFFSET && !isClosed() && isMaximized()) {
      minimize();
    }
    setEnableTouch(!(slideOffset <= MIN_SLIDE_OFFSET));
    ViewHelper.setX(this, width - slide);
  }

```